ros/src/system/emergency_handler/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(emergency_handler)
+
+add_compile_options(-std=c++11)
+
+find_package(
+  catkin REQUIRED COMPONENTS
+  autoware_system_msgs
+  autoware_health_checker
+  roscpp
+  std_msgs
+)
+
+catkin_package(
+  LIBRARIES emergency_handler system_status_filter
+  CATKIN_DEPENDS
+  autoware_system_msgs
+  autoware_health_checker
+  roscpp
+  std_msgs
+)
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_library(system_status_filter lib/libsystem_status_filter.cpp)
+target_link_libraries(system_status_filter
+  ${catkin_LIBRARIES}
+)
+add_dependencies(system_status_filter ${catkin_EXPORTED_TARGETS})
+
+add_library(emergency_handler lib/libemergency_handler.cpp)
+target_link_libraries(emergency_handler
+  system_status_subscriber
```

Comment:
@yn-mrse is _system_status_subscriber_ supposed to be built in this package? If so, it is not being built, hence the CI failures

Reply:
Thank you, sgermanserrano.
`system_status_subscriber` library is external dependence; in `autoware_health_checker` package.
Am I misunderstanding how to install libraries?

Reply:
@yn-mrse #1992 fixes the CI problems